packages/foreman/foreman-installer/foreman-installer.spec

```diff
@@ -30,13 +30,28 @@ BuildRequires: puppet-agent-puppet-strings < 2
 %description
 Complete installer for The Foreman life-cycle management system based on Puppet.
 
+%package katello
+Summary: Katello installer bits
+Group: Applications/System
+Provides: katello-installer-base < 3.11
```

Comment:
Why do we need this?

Reply:
https://docs.fedoraproject.org/en-US/packaging-guidelines/#renaming-or-replacing-existing-packages also lists a Provides but with a `=`. Given we did bump it to 3.11 in nightly, I'm unsure which exact version to use.